_includes/pbc/all/install-features/202410.0/install-glue-api/install-the-payments-glue-api.md

```diff
@@ -173,12 +174,100 @@ class GlueApplicationDependencyProvider extends SprykerGlueApplicationDependency
 
         return $resourceRelationshipCollection;
     }
+    
+     /**
+     * {@inheritDoc}
+     *
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceRoutePluginInterface>
+     */
+    protected function getResourceRoutePlugins(): array
+    {
+        return [
+            new PaymentCustomersResourceRoutePlugin(),
+        ];
+    }
 }
 ```
 
 {% info_block warningBox "Verification" %}
 
 To verify `PaymentMethodsByCheckoutDataResourceRelationshipPlugin` is activated, send a POST request to `https://glue.mysprykershop.com/checkout-data?include=payment-methods` and make sure that `checkout-data` resource has a relationship to the `payment-methods` resources.
+To verify `PaymentCustomersResourceRoutePlugin` is activated, send a POST request to `https://glue.mysprykershop.com/payment-customers` and make sure that you get a response with customer data.
+
+An example request for `https://glue.mysprykershop.com/payment-customers`:
+The `customerPaymentServiceProviderData` field is a placeholder for the data that should be returned by the PSP widget.
+
+```json
+{
+  "data": {
+    "type": "payment-customers",
+    "attributes": {
+      "payment": {
+        "paymentMethodName": "dummyPayment",
+        "paymentProviderName": "dummyPaymentProvider"
+      },
+      "customerPaymentServiceProviderData": "customerPaymentServiceProviderData"
```

Comment:
this example with dummyPayment most probably does not return the correct result. So maybe we can rely on the validation errors or some error that it's not supported by dummy payment